_src/static/sass/utilities/_variables.scss

```diff
@@ -233,6 +230,17 @@ $font-weights: (
     x-bold:         800
 );
 
+// typography: letter-spacing
+$letter-spacing: (
+    normal:         0,
+    tightest:       -3,
+    tighter:        -2,
+    tight:          -.095rem,
+    loose:          -.075rem,
+    looser:         -.025rem,
+    loosest:        3
+);
+
```

Comment:
Aware of the... naming. It's unused at this point. If we decide to keep it we can re-think the map variables.